content/en/seachat/seachat-manual/02-create-agent/03-advanced-settings/01-agent-information.md

```diff
@@ -125,4 +124,23 @@ In addition to the style settings, you can also customize the chat settings. Thi
 
 ## Advanced Settings
 
-We suggest you to refer to the [Advanced Settings](/seachat/seachat-manual/02-create-agent/05-advanced-settings) tutorial to learn more about the advanced features available in SeaChat. For feature such as Agent Memory and Retrieval Augmented Generation (RAG), you can find them in the **Advanced Settings** section under **Agent Information**.
\ No newline at end of file
+We suggest you to refer to the [Advanced Settings](/seachat/seachat-manual/02-create-agent/05-advanced-settings) tutorial to learn more about the advanced features available in SeaChat. For feature such as Agent Memory and Retrieval Augmented Generation (RAG), you can find them in the **Advanced Settings** section under **Agent Information**.
+
+## :dart: Troubleshooting
+
+### How to change my agent's settings in channel and integration?
```

Comment:
i'm a little confused about this part -- is this from the customer who wants a different language for "live agent" for Line? This answer doesn't seem to address the problem